frontend/docker-compose.yml

```diff
@@ -7,4 +7,4 @@ services:
     container_name: angular-ui
```

Comment:
This file's content is to be later moved in parent docker-compose.yml, right?

Reply:
Yes it will be moved and refactored a bit to docker-compose version 2.